README.md

```diff
@@ -7,7 +7,7 @@ AutoMapper.Extensions.EnumMapping
 
 ### Summary
 
-The AutoMapper.Extensions.EnumMapping library gives you control about your enum values mappings. Normally enums are automatically mapped bij AutoMapper as an integer, but you have no control about custom mappings. It is possible to create a custom type converter for every enum.
+The AutoMapper.Extensions.EnumMapping library gives you control about your enum values mappings. Normally, enums are automatically mapped by AutoMapper by performing a case-insensitive TryParse, before falling back to mapping by using the internal-integer value, but you have no control about custom mappings. It is possible to create a custom type converter for every enum.
```

Comment:
Ok, it is `TryParse`, but on what? :) Maybe say smth like map by string values? Then fallback to mapping by the underlying numeric value.